bin/burden

```diff
@@ -4063,7 +4096,7 @@ for i in `ls -d *`; do
 			date=`date`
 			echo "$data: $date tests completed: ${tests}, tests left ${tests_left}" >> /home/zathras_log/spot_fails
 		fi
-		sed "s/--use_spot/--no_spot_recover/g" $gl_top_dir/$file_to_use | sed "s/${tests}//g" > $gl_top_dir/rerun
+		sed "s/--use_spot 1/--no_spot_recover/g" $gl_top_dir/$file_to_use | sed "s/${tests}//g" > $gl_top_dir/rerun
```

Comment:
It looks like you're making "--use_spot 1" as an alias for "--no_spot_recover".  Why not just use "--no_spot_recover" where it's needed or if you have to make "--use_spot" take args at least make them symbolic rather than magic numbers?